StringCalculatorKata/StringCalculatorKata/StringCalculator.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections;
+using System.Linq;
+
+namespace StringCalculatorKata
+{
+    public class StringCalculator
+    {
+        private string _numbers;
+        private List<char> _delimiters = new List<char>() { ',', '\n' };
+
+        public int add(string numbers)
+        {
+            _numbers = numbers;
```

Comment:
`Add` shouldn't save the input anywhere really. This method should be more abstract, and you can make it as a static method actually, no need to initialize an object from `StringCalculator` to use the methods it has.

Reply:
I made some changes and pushed them
you can check now ;)